src/python/espressomd/analyze.py

```diff
@@ -192,6 +192,16 @@ class Analysis(ScriptInterfaceHelper):
         (N,) array_like of :obj:`int`
             The neighbouring particle ids.
 
+    particle_neighbor_pids()
+        Get a dictionary, where each key-value-pair represents a particle
+        and it's respective neighbors.
```

Comment:
```suggestion
        Get a list of all short-range neighbors for each particle.
```
To avoid repetition with the return paragraph.